src/main/java/net/dries007/tfc/common/blockentities/TickCounterBlockEntity.java

```diff
@@ -38,6 +38,23 @@ public long getTicksSinceUpdate()
         return Calendars.SERVER.getTicks() - lastUpdateTick;
     }
 
+    /**
+     * Use resetCounter over this. Only use if you know what you're doing.
+     */
+    public void setLastUpdateTick(long tick)
+    {
+        lastUpdateTick = tick;
+        setChanged();
+    }
+
+    /**
+     * Use getTicksSinceUpdate over this. Only use if you know what you're doing.
+     */
```

Comment:
These are bad comments. If these functions have strange or unexpected behavior, *document that behavior*. This is like code gatekeeping lol.